src/openrct2/windows/footpath_scenery.c

```diff
@@ -0,0 +1,484 @@
+#pragma region Copyright (c) 2014-2016 OpenRCT2 Developers
```

Comment:
Changed it to tabs, I had applied the clang-format with the config file in the repo and that converted to spaces. So different style for C and C++ files? That's a bit odd.